GaelAlburo/backend-project
# Add requirements.txt for dependency management

Description:
This PR adds a `requirements.txt` file to the project, enabling consistent dependency management across environments.

Changes:
- Added a `requirements.txt` file generated using pip freeze.

Impact:
- Facilitates collaboration by standardizing the dependencies for the project.  
- Simplifies the process for setting up and deploying the project.
- Ensures compatibility between team members working on different operating systems.

Testing:
  1. Clone the repository.  
  2. Create and activate a virtual environment.  
  3. Run `pip install -r requirements.txt` to verify that the dependencies are installed successfully.
